geode-core/src/distributedTest/java/org/apache/geode/internal/cache/control/RebalanceOperationDistributedTest.java

```diff
@@ -566,7 +567,8 @@ public void testRecoverRedundancyBalancingIfCreateBucketFails() {
 
       PartitionedRegion origRegion = (PartitionedRegion) cache.getRegion("region1");
       PartitionedRegion spyRegion = spy(origRegion);
-      PRHARedundancyProvider redundancyProvider = spy(new PRHARedundancyProvider(spyRegion));
+      PRHARedundancyProvider redundancyProvider =
+          spy(new PRHARedundancyProvider(spyRegion, mock(InternalResourceManager.class)));
```

Comment:
Assuming this ok because we don't care what InternalResourceManager is doing in terms of the Rebalance Operation